src/main/scala/vsys/api/http/contract/ContractApiRoute.scala

```diff
@@ -91,6 +93,16 @@ case class ContractApiRoute (settings: RestAPISettings, wallet: Wallet, utx: Utx
     }
   }
 
+  private def typeFromBytes(bytes: Array[Byte]): String = {
+    if (bytes sameElements ContractPermitted.contract.bytes.arr) {
+      "TokenContractWithSplit"
```

Comment:
LGTM.  Ideally we define enum for contract types.

Reply:
Sure, got it.